pkg/controller/workload/job/job_controller.go

```diff
@@ -44,6 +44,7 @@ import (
 )
 
 var (
+	ownerKey          = ".metadata.controller"
```

Comment:
this is an odd key. Generally it should match the jsonpath of where the value is found.
But in reality it doesn't matter.

Still, this should probably reference job in some way. What about `.metadata.ownerReferences[Job]`?

Reply:
Done, but I'm think that eventually, rather than requiring the frameworks to implement it based on GVK.